apps/judicial-system/backend/migrations/20250318105658-update-case-files.js

```diff
@@ -0,0 +1,101 @@
+'use strict'
+
+const replaceEnum = require('sequelize-replace-enum-postgres').default
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    return queryInterface.sequelize
+      .transaction((t) =>
+        Promise.all([
+          queryInterface.addColumn(
```

Comment:
Is a new field really needed? It seems to be used the same way as submittedBy once set. Are we somewhere using the fact that the court uploaded on behalf of the given submitter?